### DISCUSSION TITLE
Handling field → protected method conversions?

### DISCUSSION THREAD

**Discussion by zenfyrdev** [Sinytra/Connector#51] [General]
Hi, while testing one of my mods, I noticed that fields patched by Forge into protected methods don't seem to get transformed by Connector.

Example: [MobBucketItem](https://github.com/MinecraftForge/MinecraftForge/blob/1.20.x/patches/minecraft/net/minecraft/world/item/MobBucketItem.java.patch). The `entityType` field becomes `getFishType()`. I wanted to submit a PR with the fields added to `FieldToMethodTransformer#REPLACEMENTS`, but I realized that they are protected and can't be widened. 

Are there any plans to handle collisions like this?

The easiest way would probably be to create a public version like this `public <return type> connector$<method name>(<params>)` and redirect fields to it, but you probably have a better idea. 😅

**Comment by Su5eD** [comment]
Yeah, we can handle those easily. I'll create an issue to keep track of this.
